components/concept/storage/README.md

```diff
@@ -5,7 +5,6 @@ The `concept-storage` component contains interfaces and abstract classes that de
 This abstraction makes it possible to build components that work independently of the storage layer being used.
 
 Currently two store implementations are available:
```

Comment:
Can you update this comment as well? "Currently a single store implementation is available:"

Reply:
Ohh yes, such a silly one. Will do that!